TaskLayer/MyFileManager.cs

```diff
@@ -1,35 +1,20 @@
 ﻿using EngineLayer;
-using IO.MzML;
 using IO.Mgf;
-
-#if NETFRAMEWORK
-
-using IO.Thermo;
-
-#else
-#endif
-
+using IO.MzML;
 using MassSpectrometry;
 using System;
 using System.Collections.Generic;
-using System.Diagnostics;
 using System.IO;
-using System.Linq;
 using System.IO.Compression;
+using ThermoRawFileReader;
```

Comment:
is there no option to continue using thermofilereader?

Reply:
not in MM. the code to use MSFileReader still exists in mzLib but isn't referenced in MM or FlashLFQ anymore. it can still be referenced by editing the code, but it's not a user-facing option